src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow/SimulationElbow.java

```diff
@@ -0,0 +1,79 @@
+package edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
+
+import com.revrobotics.CANSparkMax;
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+
+import static edu.greenblitz.tobyDetermined.RobotMap.TelescopicArm.Elbow.Simulation.SIM_PID;
+
+public class SimulationElbow implements IElbow {
+    SingleJointedArmSim elbowSim;
+    private double appliedVoltage;
+
+    public SimulationElbow() {
+        elbowSim = new SingleJointedArmSim(
+                DCMotor.getNEO(RobotMap.TelescopicArm.Elbow.Simulation.MotorSimulationConstants.NUMBER_OF_MOTORS),
+                RobotMap.TelescopicArm.Elbow.Simulation.MotorSimulationConstants.GEAR_RATIO,
+                SingleJointedArmSim.estimateMOI( RobotMap.TelescopicArm.Extender.EXTENDED_LENGTH, 6), //-> 1.0 / 3.0 * mass * length^2,
```

Comment:
What is 6?

Reply:
its the mass of the extender. now its in a constant